doc/manual/using_subsystems.rst

```diff
@@ -0,0 +1,194 @@
+Datasets and subsystems
+=======================
+
+Beyond running single experiments, or basic use of master, client, and dashboard, ARTIQ supports a much broader range of interactions between its different components. These integrations allow for system control which is extensive, flexible, and dynamic, with space for different workflows and methodologies depending on the needs of particular sets of experiments. We will now explore some of these further tools and systems.
+
+.. note::
+
+    This page follows up directly on :doc:`getting_started_mgmt`, but discusses a broader range of features, most of which continue to rest on the foundation of the management system. Some sections (datasets, the browser) are still possible to try out using your PC alone; others (MonInj, the RTIO analyzer) are only meaningful in relation to some kind of real-time hardware.
```

Comment:
change "in relation to some kind of real-time hardware" to "with a core device and relevant peripherals"

Reply:
Definitely awkward phrasing, I'll rephrase, but I think there's some grace to the generality of 'real-time hardware', after all the systems _work_ without any peripherals.

Reply:
Could be resolved by adding an "optionally", no? IMO describing the concrete product that the user is interfacing with rather than the less specific "real-time hardware" is preferable.